jbpm-event-emitters/jbpm-event-emitters-elasticsearch/src/test/java/org/jbpm/event/emitters/elasticsearch/ElasticSearchEventEmitterTest.java

```diff
@@ -308,6 +325,91 @@ protected ExecutorService buildExecutorService() {
         assertThat(responseCollector.get(0)).isEqualToNormalizingNewlines(expectedResult);
     }
     
+    @Test
+    public void testTaskInstanceThroughEmitterIgnoreNull() throws Exception {
+
+        System.setProperty("org.jbpm.event.emitters.elasticsearch.ignoreNull", "true");
+        ElasticSearchEventEmitter emitter = null;
+        try {
+            SimpleDateFormat sdf = new SimpleDateFormat(dateFormatStr);
+            // sample date must match one set in the expected result file
+            Date sampleDate = sdf.parse("2018-10-23");
+
+            String expectedResult = read(this.getClass().getResourceAsStream("/testTaskInstanceThroughEmitterIgnoreNull.json"));
+
+            Task taskInstance = Mockito.mock(Task.class);
+            InternalTaskData taskData = Mockito.mock(InternalTaskData.class);
+            User user = Mockito.mock(User.class);
+            InternalPeopleAssignments peopleAssignments = Mockito.mock(InternalPeopleAssignments.class);
+
+            when(peopleAssignments.getBusinessAdministrators()).thenReturn(Collections.emptyList());
+            when(peopleAssignments.getExcludedOwners()).thenReturn(Collections.emptyList());
+            when(peopleAssignments.getPotentialOwners()).thenReturn(Collections.emptyList());
+
+            when(user.getId()).thenReturn("john");
+
+            Map<String, Object> variables = new HashMap<>();
+            variables.put("initiator", "john");
+            variables.put("variable", 123);
```

Comment:
There is no `null` value set in this test case to make sure the null values are ignored.

I would set for instance, this to `null` and remove this field from the expected output in the json file `testTaskInstanceThroughEmitterIgnoreNull.json`